core/openapi.yaml

```diff
@@ -343,58 +343,41 @@ components:
           type: number
           format: double
           example: 0.25
-        timetable_gamma:
+        gamma:
           type: number
           format: double
           example: 0.5
-        tractive_effort_curve:
-          type: array
-          items:
-            type: object
-            properties:
-              speed:
-                type: number
-                format: double
-                minimum: 0
-              max_effort:
-                type: number
-                format: double
-                minimum: 0
-            example:
-              speed: 20
-              max_effort: 400000
-    RollingResistanceDavis:
```

Comment:
please make sure the file is still valid using https://editor.swagger.io/
RollingResistanceDavis cannot be removed while still used